### ISSUE TITLE
How to Silently Uninstall the Agent

### ISSUE THREAD

**Issue by smontgomery345** [basildane/WakeOnLAN#219]
Hello, is there a silent uninstall Switch for the Agent ?

Thanks

**Comment by basildane** [comment]
Yes, the installer uses Innosetup, which has a /SILENT, /VERYSILENT options.
However, I have not tested them for this.  
[https://jrsoftware.org/ishelp/](Innosetup)
